lib/AgentSDK.js

```diff
@@ -311,7 +316,11 @@ class SDK extends Events {
                 this.jar = cookies;
                 this.csrf = data.csrf;
                 this.token = data.bearer;
-                this.startPeriodicRefreshSession();
+
+                // start the session token refresh
+                if (this.conf.conf.refreshSessionToken) {
```

Comment:
@asicath Is this supposed to be `this.conf` instead of `this.conf.conf`?

Reply:
yup, fixed. Just testing to see if you read it ;)